index.html

```diff
@@ -0,0 +1,176 @@
+                    </svg></a>
+
+            </li>
+            <li class="nav-item"><a href="href">English</a></li>
+            <li class="My-Page"><a href="#">My Page</a></li>
+            <li><a href="#" class="">Logout</a></li>
+            <li class="offset-1"></li>
+        </ul>
+
+        <nav class="nav-bar">
+            <svg fill="#FFFFFF" id="btn-close" class="btn-close jus" xmlns="http://www.w3.org/2000/svg"
+                viewBox="0 0 50 50" width="25px" height="25px">
+                <path
+                    d="M 11.5 11 C 11.372 11 11.243984 11.048984 11.146484 11.146484 C 11.049484 11.244484 11 11.372 11 11.5 C 11 11.628 11.048484 11.755516 11.146484 11.853516 L 24.292969 25 L 11.146484 38.146484 C 10.951484 38.341484 10.951484 38.658516 11.146484 38.853516 C 11.244484 38.950516 11.372 39 11.5 39 C 11.628 39 11.755516 38.951516 11.853516 38.853516 L 25 25.707031 L 38.146484 38.853516 C 38.341484 39.048516 38.658516 39.048516 38.853516 38.853516 C 39.048516 38.657516 39.049516 38.342484 38.853516 38.146484 L 25.707031 25 L 38.853516 11.853516 C 39.048516 11.658516 39.048516 11.341484 38.853516 11.146484 C 38.657516 10.951484 38.342484 10.950484 38.146484 11.146484 L 25 24.292969 L 11.853516 11.146484 C 11.756016 11.048984 11.628 11 11.5 11 z" />
+            </svg>
+            <a href="index.html" class="navbar-brand text-dark"><img src="image/mdslogo.png" alt="logo"> </a>
+            <ul class="nav-menu">
+
+                <li><a href="index.html" class="mob-home">Home</a></li>
+                <li><a href="about.html">About</a></li>
+                <li><a href="#product" >Product</a></li>
+                <li><a href="#partner">Partner</a></li>
+                <li><a href="about.html#past-event" class="nav-menu-cc">Events</a></li>
+            </ul>
+
+        </nav>
+    </header>
+
+    <main>
+        <section class="headline-section">
+
+            <svg fill="#000000" class="menu-icon" id="menu-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 50 50"
+                width="100px" height="100px">
+                <path
+                    d="M 0 7.5 L 0 12.5 L 50 12.5 L 50 7.5 Z M 0 22.5 L 0 27.5 L 50 27.5 L 50 22.5 Z M 0 37.5 L 0 42.5 L 50 42.5 L 50 37.5 Z" />
+            </svg>
+            <p class="headline-greeting">Hello! Product Designers</p>
+            <h1 class="headline-title">Global association for creating quality softwrae products 2022
+            </h1>
```

Comment:
Rather than giving fixed width you can use `<br />` to break the text that way it will be easy to make it responsive. Also, to match the design let's make this text bold and increase it's font-size.

| Your Design | Design provided |
|-------------|---------------|
| <img width="1010" alt="image" src="https://user-images.githubusercontent.com/34578631/181297123-efd3647f-0032-41d2-b409-a68898f76254.png"> | <img width="794" alt="image" src="https://user-images.githubusercontent.com/34578631/181297171-b1153f7d-48a0-45c9-a005-81ea5fd2dccc.png"> |